contrib/p5_wild_wyss/src/logger/logger.js

```diff
@@ -39,38 +39,39 @@ export {
  * Furthermore, each log statement has a context. The log message will only be logged, if the globalContext
  * (set with {@link setGlobalContext}) has the same prefix as the log message's context.
  *
- * The result of the callback function {@link FormatLogMessage} will be logged using the given {@link AppendCallback AppendCallbacks}.
+ * The result of the callback function {@link FormatLogMessage} will be logged using the given {@link AppendCallback AppendCallback's}.
  *
  * What's the difference between loggerLevel vs loggingLevel:
  * loggerLevel is the level of the respective logger
  * loggingLevel is the level at which logging is currently taking place.
  *
  * @function
- * @pure if the parameters "appender" of type {@link AppendCallback[]} and msgFormatter of type {@link FormatLogMessage} are pure.
+ * @pure if the {@link AppendCallback AppendCallback's} in the appender list and
+ *       the parameter msgFormatter of type {@link FormatLogMessage} are pure.
  * @type    {
  *               (loggerLevel:      LogLevelType)
- *            => (appendCallbacks:  AppendCallback[])
  *            => (context:          String)
  *            => (formatMsg:        FormatLogMessage)
  *            => (msg:              LogMeType)
  *            => churchBoolean
  *          }
  * @private
  * @example
- * const log = logger(LOG_DEBUG)(() => [Appender()])("ch.fhnw")(_context => _level => id);
+ * const log =  logger(LOG_DEBUG)("ch.fhnw")(_context => _level => id);
  * log("Andri Wild");
  * // logs "Andri Wild" to console
  */
-const logger = loggerLevel => appendCallbacks => context => formatMsg => msg =>
-LazyIf(
+const logger = loggerLevel  => context => formatMsg => msg =>
+  LazyIf(
       messageShouldBeLogged(loggerLevel)(context)
-    )
-    (Then(() =>
-      appendCallbacks
-        .map(     append      => append(formatMsg(context)(loggerLevel(snd))(evaluateMessage(msg))))
-        .reduce(  (acc, cur)  => and(acc)(cur), True)) // every() for array of churchBooleans
-    )
-    (Else(() => False));
+  )
+  (Then(() =>
+        appenderList
+            .map(appender =>
+                appender[loggerLevel(snd).toLowerCase()](formatMsg(context)(loggerLevel(snd))(evaluateMessage(msg))))
```

Comment:
` appender[loggerLevel(snd).toLowerCase()]` shouldn't we place this in a separate function? to reduce complexity of this function?